theseus_gui/src/store/search.js

```diff
@@ -0,0 +1,183 @@
+import { defineStore } from 'pinia'
+import generated from '@/generated'
+
+export const useSearch = defineStore('searchStore', {
+  state: () => ({
+    searchResults: [],
+    searchInput: '',
+    totalHits: 0,
+    currentPage: 1,
+    pageCount: 1,
+    offset: 0,
+    filter: 'Relevance',
+    categories: {},
+    loaders: {},
+    environments: {
+      client: false,
+      server: false,
+    },
+    activeVersions: [],
+    openSource: false,
+    limit: 20,
+  }),
+  actions: {
+    initFacets() {
+      var categories = generated.categories.filter((cat) => cat.project_type === 'modpack')
+
+      var loaders = generated.loaders.filter((loader) => {
+        // Some loaders don't have any supported proj types set
+        if (!loader.supported_project_types) return
```

Comment:
which ones? all of them should

Reply:
![image](https://user-images.githubusercontent.com/30800863/229312767-60fa09f0-9c49-4a2d-9b54-93a0d3df234e.png)